content/ui/enforce-onboarding-quickstart.md

```diff
@@ -26,16 +26,10 @@ To install `chainctl`, we’ll use the `curl` command to pull the application do
 curl -o chainctl "https://dl.enforce.dev/chainctl_$(uname -s)_$(uname -m)"
```

Comment:
@katjuell I made a number of updates to the quickstart guide based on changes to the onboarding doc